lib/caesar.rb

```diff
@@ -1,6 +1,25 @@
 class Caesar
-  attr_reader :key
 
-  def initialize
+  attr_reader :key, :eng_to_cipher
+
+  def initialize(eng_to_cipher)
+    @eng_to_cipher = eng_to_cipher
```

Comment:
Don't name variables the same as methods. because you have an attr_reader called eng_to_cipher and another method eng_to_cipher, which one will get called? One of them is being overwritten by the other.